content/common/contributing-docs.md

```diff
@@ -233,15 +238,63 @@ npx markdownlint-cli2 content/spin/install.md \"#node_modules\"
 
 ### 6.2 Generating Indexing For Your Content
 
-The documentation site implements in-house search and therefore it is recommended to generate a new index each time you update and push changes to the developer documentation repository. This is done simply by using the following command; in the developer directory:
+The developer documentation site implements in-house search and therefore it is recommended to generate a new index each time you update and push changes to the developer documentation repository. This is done simply by using the following command; in the developer directory:
 
 <!-- @selectiveCpy -->
 
 ```bash
 $ npm run build-index
 ```
 
-### 6.3 How To Properly Edit CSS Styles
+### 6.3 Increasing Search Visibility For Your Content
+
+The in-house search functionality is based on the indexing of individual words in each markdown file, which works well most of the time. However, there are a couple of scenarios where you _may_ want to deliberately increase search visibility for your content.
+
+**Word Mismatch**
+
+The words in your markdown file (that may help a user) may not be words that a user is actually searching for. For example, you may write about "different HTTP listening options" whereas a user may only ever try to find that specific content using a search phrase like "request alternate port". In this case (if/when you are aware of user search patterns), it is recommended that you reword your content to include the potential user's search phrase. If this is not possible (there is a mismatch between words in your content and works a user searches for) then you can alternatively use the `keywords` string in the `[extra]` section of your document's frontmatter. For example, the following show frontmatter that helps a user find your page (when using `port` in their search phrase):
```

Comment:
I yield to no-one in my love of parentheses, but I found the number of bracketed clauses in this paragraph broke up the reading experience for me.  I wonder if we can reorganise some sentences to explain the use case without the need for asides?

Reply:
Thank you, I have reworded this paragraph in the next commit/push.